lib/src/models/subreddit_moderation.dart

```diff
@@ -382,17 +384,17 @@ class SubredditSettings {
   bool get allowPostCrossposts => _data['allow_post_crossposts'];
   set allowPostCrossposts(bool x) => _data['allow_post_crossposts'] = x;
 
-  // TODO(bkonyi): figure out what this is.
-  // get spamComments => _data['spam_comments'];
+  // Filter strength for comments.
+  get spamComments => _data['spam_comments'];
 
-  // TODO(bkonyi): figure out what this is.
-  // get spamSelfposts => _data['spam_selfposts'];
+  // Filter strength for self posts.
+  get spamSelfposts => _data['spam_selfposts'];
 
   String? get submitTextLabel => _data['submit_text_label'];
   set submitTextLabel(String? x) => _data['submit_text_label'] = x;
 
   // TODO(bkonyi): we might want to use a color class for this.
-  // get keyColor => _data['key_color];
+  // get keyColor => _data['key_color'];
```

Comment:
Need confirmation on what needs to be done. Cant just the `color` package be used for this(i thought it was already being used here).

Reply:
Yeah, this is probably a really old TODO. `package:color` should be fine to use here.